crane/app_util.py

```diff
@@ -107,9 +114,16 @@ def wrapper(image_id, *args, **kwargs):
             if not repo_tuple.protected:
                 found_match = True
                 break
-            elif cert and cert.check_path(repo_tuple.url_path):
-                found_match = True
-                break
+            elif repo_tuple.protected:
```

Comment:
Looks like this should just be `else:`, or even just a separate block outside of the `if` scope like:
```
if not repo_tuple.protected:
    found_match = True
    break

value = ...
```